src/app/drag-and-drop/task/task.component.scss

```diff
@@ -0,0 +1,8 @@
+:host {
```

Comment:
Can you change this to
```
:host {
  height: min-content;
}

mat-card {
  border: 1px solid transparent;

  &.selected {
    border: 1px solid #2196f3;
  }
}
```

Because without the border, the card changes its size when selected.